apps/src/code-studio/components/ShareDialog.jsx

```diff
@@ -37,26 +47,137 @@ var ShareDialog = React.createClass({
+    var twitterShareUrl = "https://twitter.com/intent/tweet?url=" + this.props.encodedShareUrl +
+                          "&amp;text=Check%20out%20what%20I%20made%20@codeorg&amp;hashtags=HourOfCode&amp;related=codeorg";
+
+    var abuseStyle = {
+      border: '1px solid',
+      borderRadius: 10,
+      padding: 10,
+      marginBottom: 20
+    };
+
+    var abuseTextStyle = {
+      color: '#b94a48',
+      fontSize: 14
+    };
+
+    var abuseContents;
+    if (this.props.isAbusive) {
+      abuseContents = <AbuseError
+                          i18n={{
+                              tos: this.props.abuseTos,
+                              contact_us: this.props.abuseContact
+                            }}
+                          className='alert-error'
+                          style={abuseStyle}
+                          textStyle={abuseTextStyle}/>;
+    }
+
+    var sendToPhone;
+    if (this.state.showSendToPhone) {
+      sendToPhone = <SendToPhone
+                        channelId={this.props.channelId}
+                        appType={this.props.appType}
+                        styles={{label:{marginTop: 15, marginBottom: 0}}}
+                    />;
+    }
+
+    var advancedOptions;
+    if (this.props.appType === 'applab') {
+      advancedOptions = (
+        <AdvancedShareOptions
+            i18n={this.props.i18n}
+            onClickExport={this.props.onClickExport}
+            expanded={this.state.showAdvancedOptions}
+            onExpand={this.showAdvancedOptions}
+        />
+      );
+    }
+
     return (
       <BaseDialog useDeprecatedGlobalStyles isOpen={this.state.isOpen} handleClose={this.close}>
-        <ShareDialogBody
-          i18n={this.props.i18n}
-          icon={this.props.icon}
-          title={this.props.title}
-          shareCopyLink={this.props.shareCopyLink}
-          shareUrl={this.props.shareUrl}
-          encodedShareUrl={this.props.encodedShareUrl}
-          closeText={this.props.closeText}
-          isAbusive={this.props.isAbusive}
-          abuseTos={this.props.abuseTos}
-          abuseContact={this.props.abuseContact}
-          channelId={this.props.channelId}
-          appType={this.props.appType}
-          onClickPopup={this.props.onClickPopup}
-          onClickClose={this.close}
-          onClickExport={this.props.onClickExport}
-          />
+        <div>
+          {image}
+          <div id="project-share" className={modalClass} style={{position: 'relative'}}>
+            <p className="dialog-title">{this.props.title}</p>
+            {abuseContents}
+            <p style={{fontSize: 20}}>
+              {this.props.shareCopyLink}
+            </p>
+            <div style={{marginBottom: 10}}>
+              <input
+                  type="text"
+                  id="sharing-input"
+                  onClick={select}
+                  readOnly="true"
+                  value={this.props.shareUrl}
+                  style={{cursor: 'copy', width: 465}}/>
```

Comment:
Another thing we could do if we wanted this to better match our style guidelines is move all of these inline styles into a styles object at the top of the file.